db/pre-re/item_db.conf

```diff
@@ -1159,7 +1159,7 @@ item_db: (
 	Id: 601
 	AegisName: "Wing_Of_Fly"
 	Name: "Fly Wing"
-	Type: "IT_DELAYCONSUME"
```

Comment:
Any reason to change this?

Reply:
Skill casting items should not be of type `IT_DELAYCONSUME`.
In case of Fly Wing it wouldn't matter, but for the sake of consistency I changed it for all skill casting items.

Reply:
By saying "should not be", do you mean that they are officially consumed immediately, or that one must never set them as delay consumed, even if they're creating a custom item?

Reply:
In AEGIS they are consumed immediately after using them.
And see my note in doc/script_commands.txt:

> Important: Items which use itemskill() should be of type IT_USABLE.
>            If the item type is IT_DELAYCONSUME and ISF_IGNORECONDITIONS is set,
>            the item won't be consumed when using the item!

Using `IT_DELAYCONSUME` brings some risk.